plugins/content/config/model.schema

```diff
@@ -291,6 +291,10 @@
     },
     "themeSettings": {
       "type": "object"
+    },
+    "_themePreset": {
```

Comment:
Note: needs adding to framework.

Reply:
I've added a separate issue to this milestone, otherwise this is going to get lost https://github.com/adaptlearning/adapt_authoring/issues/2220